docs/table_design/dynamic_partitioning.md

```diff
@@ -35,16 +35,19 @@ PROPERTIES(
+| dynamic_partition.history_partition_num | No       | The number of historical partitions created by the dynamic partitions mechanism, with a default value of `0`. When the value is greater than 0, historical partitions are created in advance. From V2.5.2, StarRocks supports this parameter.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 |
+| dynamic_partition.start_day_of_week     | No       | When time_unit is WEEK, this parameter is used to specify the first day of each week.  Valid values: 1 to 7. 1 means Monday and 7 means Sunday. The default value is 1, which means that every week starts on Monday.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                |
+| dynamic_partition.start_day_of_month    | No       | When time_unit is MONTH, this parameter is used to specify the first day of each month. Valid values: 1 to 28. 1 means the 1st of every month and 28 means the 28th of every month. The default value is 1, which means that every month starts on the 1st. The first day can not be the 29th, 30th, or 31st.                                                                                                                                                                                                                                                                                                                                                                                                                                |
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/93beb37c-f33a-4d6f-a980-303c411a68c8)
incorrect parameter names. Change them to "dynamic_partition.time_unit".